dt/DTs_Designate/DFG-network-openstack-designate-17.1_director-rhel-virthost-3cont_2comp-ipv6-geneve.md

```diff
@@ -0,0 +1,64 @@
+# Deployed Topology 3_ocp_workers_2_compute_ipv6_geneve
```

Comment:
Please make it a more generic name; or in other words, if a more advanced keystone is not required and this DT is just about designate, have you considered expanding one of the DTs proposed here?
https://github.com/openstack-k8s-operators/architecture/pull/45